flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1088,9 +1152,14 @@ class DiscoverVolumesMethod(Names):
 
 @implementer(ILocalState)
 class FakeBlockDeviceDeployerLocalState(PClass):
+    """
+    This class is a terrible idea and should be thrown out ASAP to be
+    replaced by the real thing.
```

Comment:
:grey_question: No action required. My bad for letting this come in during review. I didn't think it fully through, looked ready to be killed in the next step.

Should have required this to be a more validated fake.